src/App.js

```diff
@@ -1,17 +1,28 @@
 import React, { useState } from 'react'
-import BylawsText from './BylawsText'
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom'
 import Header from './Header'
+import BylawsText from './BylawsText'
 import QuestionsForm from './QuestionsForm'
+import Start from './Start'
 
 function App() {
   const [answers, setAnswers] = useState([])
 
   return (
-    <>
+    <Router>
```

Comment:
I would move the Router to your index.js:
`<Router>
      <App />
    </Router>`

Reply:
Thanks for the review :-) I moved the Router to the index.js